### PR TITLE
Add an example of how to fast deloy Gitea on Sealos

### PR DESCRIPTION
**Pull request by a497625414**
### Summary
:memo::octocat::rocket:

Add a new document for installing Gitea on SealOS. The document `docs/4.0/docs/examples/gitea/install-gitea.md` explains how to use SealOS to deploy and access a self-hosted Git service.

> _`Gitea` on `SealOS`_
> _Self-hosted Git in Chinese_
> _A guide for autumn_

### Walkthrough
* Add a new markdown file that provides a quick guide on how to install Gitea on SealOS ([link](https://github.com/labring/sealos/pull/3338/files?diff=unified&w=0#diff-9627a8c91d4173c94d2aa806cb66fef8db3410fdeeb9f0fd6fe33de5ae36f399L1-R57))

### CODE REVIEW

File: docs/4.0/docs/examples/gitea/install-gitea.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# 快速安装Gitea
+
+Gitea 是一个自己托管的Git服务程序。他和GitHub, Bitbucket or Gitlab等比较类似。Gitea的首要目标是创建一个极易安装，运行非常快速，安装和使用体验良好的自建 Git 服务。
+
+## 步骤 1：打开 「应用管理」 应用
+
```
Comment (fanux) [inline docs/4.0/docs/examples/gitea/install-gitea.md:6]: 登录 [sealos](https://cloud.sealos.io)

File: docs/4.0/docs/examples/gitea/install-gitea.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- 网络配置：
+
+  - 容器暴露端口：3000
+  - 外网访问：开启
+
+  ![](images/gitea-4.png)
+
+- 高级配置：
+  - 自定义本地存储，持久化 Appsmith 的数据（推荐 1 G）。
```
Comment (bxy4543) [inline docs/4.0/docs/examples/gitea/install-gitea.md:38]: appsmith？

**Comment by bxy4543** [comment]
If possible, you can use the sealos database as the database of gitea

File: docs/4.0/docs/examples/gitea/install-gitea.md (hunk 2 above)
Comment (a497625414) [reply docs/4.0/docs/examples/gitea/install-gitea.md:38]: I'll correct it

**Comment by cuisongliu** [comment]
fetch main code and rebase it.

File: docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
+# 快速安装 Gitea
+
+Gitea 是一个自己托管的 Git 服务程序。他和 GitHub, Bitbucket or Gitlab 等比较类似。Gitea 的首要目标是创建一个极易安装，运行非常快速，安装和使用体验良好的自建 Git 服务。
```
Comment (nowinkeyy) [inline docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md:3]: image

I think this is better.

File: docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+## 步骤 5：部署应用
+
+- 点击「部署应用」开始部署应用。
+
+![](../images/gitea-6.png)
+
+## 步骤6：配置数据库
+
+- #### 通过 Sealos 为 Gitea 配置Mysql数据库
```
Comment (nowinkeyy) [inline docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md:56]: Missing spaces between Chinese and English.

File: docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+  ![](../images/gitea-3.png)
+
+- 网络配置：
+
+  - 容器暴露端口：3000
+  - 外网访问：开启
+
+  ![](../images/gitea-4.png)
+
+- 高级配置：
```
Comment (nowinkeyy) [inline docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md:43]: There should be a blank line between lines 43 and 44.

File: docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+  - 集群类型：mysql
+  - 数据库版本：ac-mysql-8.0.30
+  - CPU（推荐）：1 Core
+  - 内存（推荐）：1 G
+
+  
+
+![](../images/gitea-11.png)
+
+- 部署成功后进入详情页面查看 Mysql 连接信息
```
Comment (nowinkeyy) [inline docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md:79]: Mysql -> MySQL

File: docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+
+![](../images/gitea-14.png)
+
+## 步骤 7：访问应用
+
+- 点击「App Launchpad」查看，当应用的 STATUS 由 Pending 变为 Running，则表明该应用已成功启动。
+
+- 当 STATUS 为 Running，即可直接访问外网地址。
+
+  ![](../images/gitea-7.png)
```
Comment (nowinkeyy) [inline docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md:105]: There seems to be an extra space at the beginning of line 105.

File: docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md (hunk 3 above)
Comment (a497625414) [reply docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md:3]: yeah,I think so

File: docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+
+- 基础配置：
+
+  - 集群类型：mysql
+  - 数据库版本：ac-mysql-8.0.30
+  - CPU（推荐）：1 Core
+  - 内存（推荐）：1 G
+
+  
+
```
Comment (nowinkeyy) [inline docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md:76]: Lines 75 and 76 seem redundant. There are several similar issues below.

**Review by nowinkeyy — COMMENTED** [review]
Note that the English version is also modified simultaneously.

File: docs/4.0/docs/examples/gitea/install-gitea.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+
+## Step 5: Deploy the application
+
+- Click「Deploy Application」 to start deploying the application.
+
+![](../images/gitea-6.png)
+
+## Step 6: Configure the database
+
+- #### Configure MySQL database for Gitea via Sealos
```
Comment (nowinkeyy) [inline docs/4.0/docs/examples/gitea/install-gitea.md:57]: Seems to be more ####.

File: docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+## 步骤 5：部署应用
+
+- 点击「部署应用」开始部署应用。
+
+![](../images/gitea-6.png)
+
+## 步骤6：配置数据库
+
+- #### 通过 Sealos 为 Gitea 配置 MySQL 数据库
```
Comment (nowinkeyy) [inline docs/4.0/i18n/zh-Hans/examples/gitea/install-gitea.md:57]: Seems to be more ####.

**Review by nowinkeyy — APPROVED** [review]
LGTM

**Review by bxy4543 — APPROVED** [review]

**Pull request merged**